docs/src/api/frequentist_regression.md

```diff
@@ -36,7 +36,6 @@ adjr2(container::FrequentistRegression)
 loglikelihood(container::FrequentistRegression)
 aic(container::FrequentistRegression)
 bic(container::FrequentistRegression)
-sigma(container::FrequentistRegression)
```

Comment:
This removes sigma from the docs, but we still provide the function in the library. Shouldn't we add a docstring to the function instead of removing it from the docs?

Reply:
Oh I was thinking you are not implementing `sigma`. Because when I ran it - it threw an error !!

Reply:
The following program runs on your doc_update branch without any errors on my computer:
```
using CRRao, DataFrames, StatsModels
df = DataFrame(y = [1,2,3], x = [1,2,3])
model = fit(@formula(y ~ x), df, LinearRegression())
println(aic(model))
println(sigma(model))
```

Did you call sigma with a bayesian model? We haven't implemented sigma for bayesian models.

Reply:
This is exactly what I did. Not sure why it threw an error !!